mdp_apps.ipynb

```diff
@@ -232,7 +232,7 @@
     }
    ],
    "source": [
-    "mdp = CustomMDP(t, rewards, terminals, init, gamma=.9)"
+    "MDP.__init__(self, init, actlist, terminals, transition_matrix, rewards, gamma=gamma)"
```

Comment:
The point of the original line was to instantiate an object of the `CustomMDP` class and store it in the `mdp` variable.
Constructor calls should be within a derived class, and I believe your constructor call should replace line number 201 instead.

Reply:
I don't think it would completely resolve the issue, but what you've done essentially avoids creating the `mdp` object and will cause problems when the `value_iteration`, `policy_iteration` and `best_policy` functions are later called, as all of them operate on the `mdp` object.

Reply:
True that. Changed Line 201 instead, so that an mdp object is still created, while resolving the error. @norvig please review